suffuse/sfs.go

```diff
@@ -34,7 +38,7 @@ func (u *Sfs) Destroy(ctx context.Context) {
 
 func (u *Sfs) Statfs(ctx context.Context, req *fuse.StatfsRequest, resp *fuse.StatfsResponse) error {
   logD("Statfs")
-  stat, err := u.RootNode.GetPath().SysStatfs()
+  stat, err := RootPath.SysStatfs()
```

Comment:
I do not have a complete understanding of the semantics here, so I'll just ask. 

Does `u.RootNode.GetPath()` have the same semantics as `RootNode`?

From a naive perspective there is a difference between the Sfs RootNode and the absolute RootPath